city_scrapers/spiders/il_board_of_examiners.py

```diff
@@ -0,0 +1,104 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlBoardOfExaminersSpider(CityScrapersSpider):
+    name = "il_board_of_examiners"
+    agency = "Illinois Board of Examiners"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.ilboe.org"]
+    start_urls = ["https://www.ilboe.org/board-information/board-meetings/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css(".minutes"):
+            meeting = Meeting(
+                title="Illinois Board of Examiners",
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
```

Comment:
This should return `BOARD` since it's the Board of Examiners. You can see more info in [the docs](https://www.ilboe.org/board-information/board-meetings/)